addons/mail/models/discuss/mail_guest.py

```diff
@@ -114,6 +114,7 @@ def _init_messaging(self):
             'current_user_settings': False,
             'hasGifPickerFeature': bool(self.env["ir.config_parameter"].sudo().get_param("discuss.tenor_api_key")),
             'hasLinkPreviewFeature': self.env['mail.link.preview']._is_link_preview_enabled(),
+            'hasMessageTranslationFeature': False,
```

Comment:
Ok for the moment to disable for guests. To see with PO whether there's value to enable or disable it.
IMO, the value is quite low. One case where it can be valuable is if you make a public discuss room as a way to chat with a community, and you could imagine the community can talk to each other even when speaking different languages, without having to rely on another website on the side.

Reply:
I'll check with PAJU.